src/atom-environment.coffee

```diff
@@ -42,7 +42,6 @@ PaneContainer = require './pane-container'
 PaneAxis = require './pane-axis'
 Pane = require './pane'
 Dock = require './dock'
-Project = require './project'
```

Comment:
Was this intentionally removed?

Reply:
This is a duplicate require, check out [L36](https://github.com/atom/atom/pull/15415/files#diff-91bf726eeaaad75efb1705745a7af175L36)